taichi/transforms/compile_to_offloads.cpp

```diff
@@ -41,6 +41,14 @@ void compile_to_offloads(IRNode *ir,
   auto print = make_pass_printer(verbose, kernel->get_name(), ir);
   print("Initial IR");
 
+  if (config.print_preprocessed_ir && start_from_ast) {
+    TI_INFO("[{}] {}:", kernel->get_name(), "Preprocessed IR");
+    std::cout << std::flush;
+    irpass::re_id(ir);
+    irpass::print(ir);
+    std::cout << std::flush;
```

Comment:
Maybe we don't need to print preprocessed IR again when `verbose == true`. WDYT @lin-hitonami

Reply:
Agree!

Reply:
makes sense.